StarRailBattleSim/src/lightcones/preservation/MomentOfVictory.java

```diff
@@ -0,0 +1,32 @@
+package lightcones.preservation;
+
+import characters.AbstractCharacter;
+import enemies.AbstractEnemy;
+import lightcones.AbstractLightcone;
+import powers.PermPower;
+import powers.PowerStat;
+import powers.TempPower;
+
+public class MomentOfVictory extends AbstractLightcone {
+
+    public MomentOfVictory(AbstractCharacter owner) {
+        super(1058, 476, 595, owner);
+    }
+
+    @Override
+    public void onEquip() {
+        this.owner.addPower(PermPower.create(PowerStat.DEF_PERCENT, 24, "Moment Of Victory Defense Boost"));
+        this.owner.addPower(PermPower.create(PowerStat.EFFECT_HIT, 24, "Moment Of Victory Effect Hit Boost"));
```

Comment:
should also add extra taunt value (I believe for this LC it's 200%). You can refer to Dance At Sunset for an example.